examples/3l-node/cli.rs

```diff
@@ -548,6 +567,52 @@ fn pay_open_invoice(
     Ok(())
 }
 
+fn list_failed_swaps(node: &LightningNode) -> Result<(), String> {
+    let failed_swaps = match node.get_unresolved_failed_swaps() {
+        Ok(s) => s,
+        Err(e) => return Err(e.to_string()),
+    };
+
+    println!(
+        "Total of {} failed swaps\n",
+        failed_swaps.len().to_string().bold()
+    );
+    for swap in failed_swaps {
+        let created_at: DateTime<Local> = swap.created_at.into();
+        println!("Failed swap created at {created_at}:");
+        println!("      Address         {}", swap.address);
```

Comment:
```suggestion
        println!("      Address:        {}", swap.address);
```

Reply:
Applied in [4daf44f](https://github.com/getlipa/lipa-lightning-lib/pull/708/commits/4daf44fb38f3852ecaa1ed59e7cb584b902f8396)